service/history/execution/mutable_state_task_generator.go

```diff
@@ -82,6 +82,10 @@ type (
 			transferTask *persistence.TransferTaskInfo,
 			targetCluster string,
 		) error
+		GenerateCrossClusterApplyParentClosePolicyTask(
```

Comment:
nit: probably add a TODO here saying ideally we should merge this method with `GenerateCrossClusterTaskFromTransferTask`?